components/movementsensor/imuwit/imu.go

```diff
@@ -200,8 +200,9 @@ func (imu *wit) Position(ctx context.Context, extra map[string]interface{}) (*ge
 	return geo.NewPoint(0, 0), 0, movementsensor.ErrMethodUnimplementedPosition
 }
 
-func (imu *wit) Accuracy(ctx context.Context, extra map[string]interface{}) (map[string]float32, error) {
-	return map[string]float32{}, movementsensor.ErrMethodUnimplementedAccuracy
+func (imu *wit) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error,
+) {
+	return nil, movementsensor.ErrMethodUnimplementedAccuracy
```

Comment:
[note to self] For the IMUs - add to the Jira ticket for implementations - return the expected angle error in accuracy. When the device is tilted in the general imuwit model over 45 degrees, that's when we can't bank on our compass heading calculation, and we should return nan or -1 with an error in the accuracy response.